vertx-pg-client/src/main/java/io/vertx/pgclient/impl/codec/PrepareStatementCommandCodec.java

```diff
@@ -30,8 +30,8 @@
 
   @Override
   void encode(PgEncoder encoder) {
-    encoder.writeParse(new Parse(cmd.sql(), cmd.statement()));
-    encoder.writeDescribe(new Describe(cmd.statement(), null));
+    encoder.writeParse(new Parse(cmd.sql(), 0L));
```

Comment:
why are we replacing `cmd.statement()` with `0L` ? if we do that it means we are actually using the un named statement. I don't get this change

Reply:
Because currently the prepared statement closing won't do anything on the wire, see https://github.com/eclipse-vertx/vertx-sql-client/blob/e8893a2680c100d39524901dce2e881ce9c124a2/vertx-pg-client/src/main/java/io/vertx/pgclient/impl/codec/CloseStatementCommandCodec.java#L28-L39

so named prepared statement won't be closed and will cause leaks. So let's make it unnamed statements now and as I said in https://github.com/eclipse-vertx/vertx-sql-client/issues/577 I will support named prepared statement for Postgres in `SqlConnection#prepare`, which could be included in another PR.

Reply:
let's complete first the other PR and then we come back to this one

Reply:
opened a PR in https://github.com/eclipse-vertx/vertx-sql-client/pull/578